apps/frontend/src/network/users/__tests__/UserProfile.test.tsx

```diff
@@ -260,6 +260,41 @@ describe('a header edit button', () => {
     );
   });
 
+  it('can open and close   personal info remaining on the tab', async () => {
```

Comment:
```suggestion
  it('remains on the same tab after closing a modal', async () => {
```
just emphasizing what the crucial thing we're testing is